app/styles/clarity-mod.css

```diff
@@ -115,4 +115,71 @@ a{
   clear: none;
   float:left;
   margin: 0 10px 0 0;
-}
\ No newline at end of file
+}
+
+/*****************************************************************************************
+The modifications below all come from customizations that the AirWatch team made to APIX.
```

Comment:
Are those styles applied to all swagger APIs?

Reply:
Yes.  They are general HTML doc cleanup so that the spacing around headings and such is a little better than what we were shipping before.  I sort of figured that this formatting was pretty arbitrary for us so we might as well go with what AirWatch wanted.